packages/cdk-graph-plugin-diagram/src/config.ts

```diff
@@ -51,6 +52,9 @@ export interface IDiagramConfigBase {
 
   /** Config for graph theme */
   readonly theme?: GraphThemeConfigProp;
+
+  /** Config for predetermined node positions */
```

Comment:
Would be good to mention that the key is a Node ID / CDK construct ID in the comment :)